app/src/layouts/common/appSidebar/helpAndService/modals/requestSupportModal/requestSupportModal.jsx

```diff
@@ -32,6 +32,8 @@ import { FieldText } from 'componentLibrary/fieldText';
 import { hideModalAction, withModal } from 'controllers/modal';
 import OpenIcon from 'common/img/open-in-new-tab-inline.svg';
 import { referenceDictionary } from 'common/utils';
+import { useTracking } from 'react-tracking';
+import { HELP_AND_SERVICE_VERSIONS_EVENTS } from 'components/main/analytics/events/ga4Events/helpAndServiceVersionsEvents';
```

Comment:
Why not to use alias `analyticsEvents` here as well?
Check other files where applicable.